test/e2e_mc/main_test.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2022 Antrea Authors
```

Comment:
```suggestion
// Copyright 2023 Antrea Authors
```
Same for all new files added

Reply:
I've updated them. Thanks!